packages/sdk/src/AMM.ts

```diff
@@ -463,16 +467,28 @@ class AMM extends Base {
       .minus({ days })
       .toSeconds()
 
-    const { totalFeesFormatted: feesEarnedToday, totalLiquidityFormatted: totalLiquidityToday } = await this.getAprForDay(endTimestamp)
+    const {
+      totalFeesFormatted: feesEarnedToday,
+      totalLiquidityFormatted: totalLiquidityToday,
+      totalVolume,
+      totalVolumeFormatted
+    } = await this.getAprForDay(endTimestamp)
```

Comment:
suggestion: rename `getAprForDay` to `getYieldStatsForDay` or more accurately `get24HourYieldStats`